Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM nikolaik/python-nodejs:python3.9-nodejs14 as build
```

Comment:
I tried using the `slim` and `alpine` image variants, but both of them encountered compilation errors when building. It's simpler to just use the regular image in this case.